packages/core/src/types.ts

```diff
@@ -78,3 +78,9 @@ export interface OutboundPackage {
   endpoint?: string
   connectionId?: string
 }
+
+export type AnyJson = boolean | number | string | null | JsonArray | JsonMap
+export interface JsonMap {
+  [key: string]: AnyJson
+}
+export type JsonArray = Array<AnyJson>
```

Comment:
This is good. I was confused a bit with `Map` in `JsonMap`, 

Could we use the following naming:
```
type JsonValue = string | number | boolean | JsonObject | JsonArray;

interface JsonObject {
  [property: string]: JsonValue;
}

interface JsonArray extends Array<JsonValue> { }
```

It's actually according to the standard https://www.json.org/json-en.html where you also have object, array and value

Reply:
Yes! This was just a copy paste from somewhere :)